phoenix-core/src/main/java/org/apache/phoenix/schema/PTableImpl.java

```diff
@@ -1586,7 +1635,12 @@ public synchronized boolean getIndexMaintainers(ImmutableBytesWritable ptr, Phoe
 
     @Override
     public PName getPhysicalName() {
+        // For views, physicalName is base table name. There might be a case where the Phoenix table is pointing to another physical table.
```

Comment:
For views, physicalName is the base table _physical table name_ or _logical_ table name?

Reply:
physical. Will update